packages/material-ui/src/FormHelperText/formHelperTextClasses.d.ts

```diff
@@ -0,0 +1,17 @@
+export interface FormHelperTextClasses {
```

Comment:
A question while I'm here: Do these need to be interfaces for module augmentation or would a `Record<FormHelperTextClassKey, string>` be sufficient?

Reply:
That's a good point, I believe we can use the same. It will also ensure that the typings on the `classses` are _correct.___

Reply:
I wonder how we should best handle this. So far the components were re-introducing this new typing. In order not to block the migration, should we handle this change in the end for all components once they are migrated? I will leave a comment on the issue for this change.

Reply:
I think that we could try to migrate one component to this approach, and if it works well, migrate them all at once? I could do it this weekend if needed.

Reply:
Sounds good to me 👍 will merge this one then and we have this https://github.com/mui-org/material-ui/issues/24405#issuecomment-769022986 for reference of what needs to be done